doc/dev/decisionrecords/UseDecisionRecords.md

```diff
@@ -0,0 +1,37 @@
+# Decision Records
+
+[TODO - Not precise] An OTP Decision Record is a justified software design choice that addresses a
```

Comment:
What does this TODO mean?

Reply:
I feel like the description of decision records is sufficiently precise as-is, so maybe the TODO could be removed. I will suggest a couple of minor style edits though.

Reply:
Thank you for your suggestions, I have applied them.